src/features/account/LogInSessions.tsx

```diff
@@ -19,6 +19,30 @@ import { Loader } from '../../common/components';
 import { useAppSelector } from '../../common/hooks';
 import { useLogout } from '../login/LogIn';
 
+const MfaStatusIndicator: FC<{ mfa: 'USED' | 'NOT_USED' | 'UNKNOWN' }> = ({
+  mfa,
+}) => {
+  if (mfa === 'USED') {
+    return (
+      <Tooltip title="MFA used">
+        <FontAwesomeIcon icon={faCheck} style={{ color: 'green' }} />
+      </Tooltip>
+    );
+  } else if (mfa === 'NOT_USED') {
```

Comment:
no need for `else` after a `return`

https://eslint.org/docs/latest/rules/no-else-return

why I started following this rule:

https://docs.astral.sh/ruff/rules/superfluous-else-return/

Reply:
I feel it makes it less readable... else makes it clear it's exclusive to the blocks above, especially for multiple if-else-ifs

Reply:
ie I agree it is "textually" (in execution) superfluous but I think it is sub-textually substantial